tools/run_tests/run_tests.py

```diff
@@ -1270,7 +1270,7 @@ def git(cmd, cwd=cwd):
     print('languages with custom make options cannot be built simultaneously with other languages')
     sys.exit(1)
   else:
-    language_make_options = next(iter(languages)).make_options()
+    language_make_options = list(set([make_option for lang in languages for make_option in lang.make_options()]))
```

Comment:
please add a comment explaining that combining the build options like this is not a clean thing to do, but for gcov it happends to work so we are using it. In all other cases, there should always be just one language.